poetry.lock

```diff
@@ -11,36 +11,22 @@ six = "*"
 
 [[package]]
 name = "aiohttp"
-version = "3.8.0"
+version = "3.7.4.post0"
```

Comment:
Can you not update the poetry.lock? There are several updates that shouldn't happen here!

This is one of them, chardet is another one, ...

Reply:
aiohttp does not work with one of the packages (I think it was Discord). This is the only thing that lets all the packages work together nicely.

Reply:
The reason why we updated to 3.8.0 is to remove the dependency for chardet which has a LGPL license

Reply:
https://github.com/Rapptz/discord.py/blob/master/requirements.txt
Looks like this is going to conflict with discord.py requirements. Since this project is no longer actively maintained should fork it with and updated requirements.txt?

Reply:
Hummm, yes, I think that's a good idea.

But in any case the discord requirements should be separated from the main terminal one as this is an "optional usage"

Reply:
Added a solution to this. User does not need to install new requirements, they just need to be present for Heroku.